### PR TITLE
Add a few tests

### PR DESCRIPTION
**Pull request by Poseyy**
Adding some tests in line with the discussion on gitter about preferably testing for int8, int16, int32, and int64 to avoid confusion. Does not *close* #100

### CODE REVIEW

File: tests/ssz/test_deserialize.py [hunk 1]
```diff
@@ -9,6 +9,9 @@
     'value, typ, data',
     [
         (5, 'int8', b'\x05'),
+        (7, 'int16', b'\x00\x07'),
+        (9, 'int32', b'\x00\x00\x00\x09'),
+        (15, 'int64', b'\x00\x00\x00\x00\x00\x00\x00\x0f'),
         (2**32-3, 'int40', b'\x00\xff\xff\xff\xfd'),
```
Comment (hwwhww) [inline tests/ssz/test_deserialize.py]: And remove the `int40` test.

File: tests/ssz/test_serialize.py [hunk 2]
```diff
@@ -9,6 +9,9 @@
     'value, typ, data',
     [
         (5, 'int8', b'\x05'),
+        (7, 'int16', b'\x00\x07'),
+        (8, 'int32', b'\x00\x00\x00\x08'),
+        (15, 'int64', b'\x00\x00\x00\x00\x00\x00\x00\x0f'),
         (2**32-3, 'int40', b'\x00\xff\xff\xff\xfd'),
```
Comment (hwwhww) [inline tests/ssz/test_serialize.py]: And remove the `int40` test.

**Comment by djrtwo** [comment]
I would add an explicit test for `int128` and `int256` because we are likely to use both of these types in the beacon chain

File: tests/ssz/test_deserialize.py [hunk 3]
```diff
@@ -12,7 +12,8 @@
         (7, 'int16', b'\x00\x07'),
         (9, 'int32', b'\x00\x00\x00\x09'),
         (15, 'int64', b'\x00\x00\x00\x00\x00\x00\x00\x0f'),
-        (2**32-3, 'int40', b'\x00\xff\xff\xff\xfd'),
+        (130, 'int128', b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x82'),
+        (225, 'int256', b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\xe1'),
```
Comment (hwwhww) [inline tests/ssz/test_deserialize.py]: Try `b'\x00' * 31 + b'\xe1`.

File: tests/ssz/test_serialize.py [hunk 4]
```diff
@@ -8,12 +8,12 @@
...
     'value, typ, data',
     [
-    #uint8, uint16, uint32, uint64
         (5, 'int8', b'\x05'),
         (7, 'int16', b'\x00\x07'),
         (8, 'int32', b'\x00\x00\x00\x08'),
         (15, 'int64', b'\x00\x00\x00\x00\x00\x00\x00\x0f'),
-        (2**32-3, 'int40', b'\x00\xff\xff\xff\xfd'),
+        (130, 'int128', b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x82'),
+        (225, 'int256', b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\xe1'),
```
Comment (hwwhww) [inline tests/ssz/test_serialize.py]: ditto.

Comment (Poseyy) [reply tests/ssz/test_serialize.py]: ah yeah, fixed, thanks!

**Pull request merged**